console/src/components/Services/EventTrigger/utils.js

```diff
@@ -140,6 +140,7 @@ FROM
     type: 'run_sql',
     args: {
       sql: runSql,
+      read_only: true,
```

Comment:
you missed checkFeatureSupport here?

Reply:
This is a single query. so nno need for version check